.env

```diff
@@ -1,4 +1,4 @@
-DB_HOSTNAME=localhost
+DB_HOSTNAME=lutedb
```

Comment:
TODO: Check if this is really needed. Ideally don't want to make yet another env file, there's enough of those already ;)

Reply:
Maybe we could create a separate .env file just for the Docker build, stored in /docker.